blockly-core/blocks/functional_examples.js

```diff
@@ -55,5 +55,23 @@ Blockly.Blocks.functional_example = {
   updateOutputType: function(outputType) {
     this.outputType_ = outputType;
     this.changeFunctionalOutput(this.outputType_);
+  },
+  /**
+   * Updates the functional examples' usage types
+   * @param {Blockly.BlockValueType} newType
+   */
+  updateUsageType: function (newType) {
```

Comment:
Oops, meant to rename this before PRing. Let me know if you have ideas. Thinking `updateInputTypes`